style-guide/index.md

```diff
@@ -3207,7 +3207,7 @@ title: "Canada.ca Content Style Guide"
     <li><cite><a href="https://www.noslangues-ourlanguages.gc.ca/en/writing-tips-plus/index-eng">Writing Tips Plus</a></cite>, Resource of the Language Portal of Canada, Government of Canada</li>
   </ul>
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Footnotes</h3>
 	<dl>
 		<dt id="fn1-dt">Footnote 1</dt>
```

Comment:
```suggestion
		<dt>Footnote 1</dt>
```

Reply:
L3216
```
<a href="#fn1-rf">
```

Reply:
L2410 has a stray open em tag
```
<p>Flesch-Kincaidreading level**<em>**: </cite>5.9</p>
```

Reply:
@delisma I'm not sure where the `em/cite` it's supposed to be as this wasn't in the scope of work I did. I would assume to remove both stray tags as it isn't in a previous mention of the same text above